File: domain/src/main/java/com/icehousecorp/maunorafiq/domain/city/City.java
package com.icehousecorp.maunorafiq.domain.city;

/**
 * Created by maunorafiq on 12/5/16.
 */

public class City {
    private final String cityName;

    public City(String cityName) {
        this.cityName = cityName;
    }

    private int ordinal;

    public String getCityName() {
        return cityName;
    }

    public int getOrdinal() {
        return ordinal;
    }

    public void setOrdinal(int ordinal) {
        this.ordinal = ordinal;
    }

    @Override
    public String toString() {
        StringBuilder stringBuilder = new StringBuilder();
        stringBuilder.append("*** City ***\n");
        stringBuilder.append("City name : " + this.getCityName() + "\n");
        stringBuilder.append("City ordinal : " + this.getOrdinal() + "\n");
        stringBuilder.append("************\n");

        return stringBuilder.toString();
    }
}

File: domain/src/main/java/com/icehousecorp/maunorafiq/domain/city/City.kt
package com.icehousecorp.maunorafiq.domain.city

/**
 * Created by maunorafiq on 12/5/16.
 */

class City(val cityName: String?) {

    var ordinal: Int = 0

    override fun toString(): String {
        val stringBuilder = StringBuilder()
        stringBuilder.append("*** City ***\n")
        stringBuilder.append("City name : " + this.cityName + "\n")
        stringBuilder.append("City ordinal : " + this.ordinal + "\n")
        stringBuilder.append("************\n")

        return stringBuilder.toString()
    }
}
